alarms/Boxcar/boxcar_alarm.py

```diff
@@ -17,7 +17,7 @@ class Boxcar_Alarm(Alarm):
 	_defaults = {
 		'pokemon':{
 			'title':"A wild <pkmn> has appeared!",
-			'long_message':"Available <a href='<gmaps>'>here</a> until <24h_time> (<time_left>).",
```

Comment:
I don't think it's necessary to edit the defaults. Since it's optional information, allow users to add this option in themselves with dynamic text substitution. Same for all the alarm files.